.github/workflows/reusable-bump-flake-lock-and-selfup.yml

```diff
@@ -65,8 +65,8 @@ jobs:
           # Needed to specify token for checkout phase, only in pushing phase is too late
           # https://github.com/orgs/community/discussions/27072#discussioncomment-3254515
           token: ${{ steps.publish-token.outputs.token }}
-      - uses: DeterminateSystems/nix-installer-action@b164ca324197520b849b91376ed19d9845526805 # v12
-      - uses: DeterminateSystems/magic-nix-cache-action@b46e247b898aa56e6d2d2e728dc6df6c84fdb738 # v7
+      - uses: DeterminateSystems/nix-installer-action@ddfca32d6f9b28188dc2d139c0786e8e69fa8757 # v12
```

Comment:
This ref points to main, not any tag :thinking: https://github.com/DeterminateSystems/nix-installer-action/tags